tests/unit/Actions.spec.ts

```diff
@@ -241,6 +245,7 @@ describe('Actions Filing Functionality', () => {
     store.state.stateModel.certifyState.certifiedBy = 'somePerson'
     store.state.stateModel.defineCompanyStep.businessContact = { email: 'someEmail', phone: '123-456-7890' }
     store.state.stateModel.currentDate = '2020/01/29'
+    store.state.stateModel.incorporationDateTime.futureEffectiveDate = new Date('2020-05-05T00:30:00.000Z')
```

Comment:
To keep the tests running for a long time, use a date far in the future, say 2099 :)

Reply:
This is a great idea. I didn't even consider this thanks :) 

So your telling me tests with an expiry date is not a good practice 😆

Reply:
👍  for today+5!

Reply:
done!